packages/@aws-cdk/cdk/lib/construct.ts

```diff
@@ -573,6 +573,13 @@ export class Construct implements IConstruct {
     return (x as any).prepare !== undefined && (x as any).validate !== undefined;
   }
 
+  /**
+   * Check whether the given construct is Taggable
+   */
+  public static isTaggable(construct: any): construct is ITaggable {
```

Comment:
I have put it in the constructor as otherwise I need to add it in a few more places. 

The Stack is not a cfnRersouce (nor I think it should be).

If I independently add it to the Stack itself, then it gets pretty messy on the validations of objects. 

If you prefer, I am happy to create a "TaggableConstructor" that extend the Constructor and has the tags in there?

Reply:
Do we even need this as a public API? Sounds like it could be a helper function that we just put near the code that consumes it

Reply:
This has just been shift/lift from the cfn-resource.ts. Changing this might imply a fairly big refactoring on the TagAspect. Probably @moofish32 can give some guidance on this, or implement it.

Reply:
Still, `isTaggable` should not be on `Construct`. How about `TagManager.isTaggable`?

Reply:
Addressed in f0a219e